app/tasks/education_form/create_daily_spool_files.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+module EducationForm
+  class CreateDailySpoolFiles
```

Comment:
Resque and DelayedJob are kind of really old school, and they serialize objects using marshal as opposed to invoking rais -- suggesting we take a look at sidekiq maybe unless we actually want to serialize objects. But Resque and DelayedJob far more error prone ime.

I think Sidekiq calls it "workers" and the workers run jobs.